packages/devextreme-schematics/src/utility/modify-json-file.ts

```diff
@@ -1,7 +1,8 @@
 import { Tree } from '@angular-devkit/schematics';
 
 export function modifyJSONFile(host: Tree, path: string, callback: (obj: any) => any) {
-  const serializedConfig = host.read(path)!.toString();
+  let serializedConfig = host.read(path)!.toString();
+  serializedConfig = serializedConfig.replace(/(\/\*[\S\s]+?\*\/\n)/, '');
```

Comment:
I have several questions about the regular expression

- Why do we need round brackets here?
- Are you sure there always be only one comment block in this file? Maybe we should use `global` flag?
- Are you sure the new line character will never be `\r\n`?

Reply:
I fixed only one scenario. But I think it is a good idea to improve reg expression